s3/s3.go

```diff
@@ -296,3 +340,80 @@ func IsS3ErrCode(err error, code string) bool {
 	}
 	return false
 }
+
+// setBucketEnc sets the encryption options on a bucket
+func (s *s3client) setBucketEnc(bucketName string) error {
+	if s.EncryptOpts == nil || !s.EncryptOpts.Enabled {
+		return nil
+	}
+
+	log.Infof("Setting Bucket Encryption: %s. (KmsKeyId: %s)", bucketName, s.EncryptOpts.KmsKeyId)
```

Comment:
should use structured logging

Reply:
Do you have an example of structure logging? I just copied that from other places in the code that follow a similar pattern.

Reply:
Found a pattern, went ahead and refactored the other log statements as well.